sys-process/acct/files/acct.initd

```diff
@@ -11,10 +11,7 @@ depend() {
 
 checkconfig() {
 	export ACCT_LOG=${ACCT_LOG:-/var/account/pacct}
-	if [ ! -e "${ACCT_LOG}" ] ; then
-		touch "${ACCT_LOG}"
-		chmod 600 "${ACCT_LOG}"
-	fi
+	checkpath -f -m 600 "${ACCT_LOG}"
```

Comment:
yes, just latest of stable and unstable, use `eshowkw` for an easy overview (i.e. `6.5.5-r3` and `6.6.2`)